lucien-ios/Assets.xcassets/backgroundPhoto.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
If you're not including this image in the PR, it should be removed and added later